torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -1164,6 +1181,8 @@ def _generate_info_dict(self, model: GraphModule) -> Dict[str, Dict]:
                     self.NUM_BATCHES_KEY: num_batches,
                     self.OUTLIER_KEY: outlier_calcs[self.OUTLIER_KEY],
                     self.SUFFICIENT_BATCHES_KEY: outlier_calcs[self.SUFFICIENT_BATCHES_KEY],
+                    self.CONSTANT_CHANNEL_KEY: outlier_calcs[self.CONSTANT_CHANNEL_KEY],
+                    self.CONSTANT_COUNTS_KEY: channel_batch_cnts,
```

Comment:
Do we need both of these? It seems if `num_constant_batches` == 0 then `has_constant_batches` will be False? Correct me if I'm misunderstanding something but the latter seems redundant.

Reply:
you are right that if `num_constant_batches == 0` then has_constant_batches is False, i just had done this to make the logic of generating the text report easier.

However, this is redundant data, so I will make this fix by removing the boolean value.